float32 cur_vel
float32 tar_vel
float32 tar_dist
bool alpha
uint8 lrc_mode
uint8 crc_mode
